bdns/dns.go

```diff
@@ -116,7 +116,7 @@ var (
 
 // DNSResolver defines methods used for DNS resolution
 type DNSResolver interface {
-	LookupTXT(string) ([]string, error)
```

Comment:
at the very least, we need to give these return variables names here so that they are documented.